apps/app/src/mocks/miragejs/orm/model/BaseModel.ts

```
@@ -0,0 +1,250 @@
+   * @returns The copy of the model attributes as a JSON object
+   */
+  toJSON(): TAttrs {
+    return { ...this._attrs };
+  }
+
+  /**
+   * Serialize the model to a string
+   * @returns The simple string representation of the model and its id
+   */
+  toString(): string {
+    let idLabel = this.id ? `(${this.id})` : '';
```

ESLint

[error] 'idLabel' is never reassigned. Use 'const' instead.

(prefer-const)